lib/views/home/skeleton_loading/home_skeleton_loading.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:shimmer/shimmer.dart';
+import 'package:kayla_flutter_ic/views/home/skeleton_loading/skeleton_placeholder.dart';
+
+class HomeSkeletonLoading extends StatelessWidget {
+  const HomeSkeletonLoading({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Shimmer.fromColors(
+      baseColor: Colors.white38,
+      highlightColor: Colors.white70,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          _buildTopSkeleton(),
+          _buildBottomSkeleton(context),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildTopSkeleton() => Container(
+        padding: const EdgeInsets.only(top: 10, right: 20, bottom: 10),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: const [
+            TextPlaceholder(
+              type: TextPlaceholderType.twoLines,
+              width: 120,
+            ),
+            SizedBox(
+              width: 40,
```

Comment:
@nmint8m You can use the `radius` properties of `CirccleAvatar` instead of wrapping it in a `SizedBox` 🙏

Reply:
It will be like this if I didn't wrap the CircleAvatar inside the SizedBox:

<img width="300" alt="Screenshot 2023-02-22 at 14 21 12" src="https://user-images.githubusercontent.com/23162627/220550535-2a927efd-babf-49d3-a2bc-58f7508c2633.png">